packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -299,18 +290,18 @@ $btn-icon-sizeXSmall-iconSize    : 18px;
     border: 0;
     background-color: transparent;
     padding: 0;
-    color: $color-link-default;
+    color: $color-content-link;
     font-weight: $font-weight-bold;
 
     &:hover {
         cursor: pointer;
-        color: $color-link-hover;
+        color: darken($color-content-link, $color-hover-01);
         background-color: transparent;
         text-decoration: underline;
     }
     &:active,
     &:focus {
-        color: $color-link-active;
+        color: darken($color-content-link, $color-active-02);
```

Comment:
I would change this to use `active-01`, as I think `active-02` is only meant to be paired with `hover-02`